### PR TITLE
Create language_pt.h

### PR DESCRIPTION
**Pull request by nfsmedeiros**
Portuguese LCD messages

### CODE REVIEW

**Comment by 3d-gussner** [comment]
@nfsmedeiros Thanks a lot, for the pull request, i will check it!
As soon i compile the 3.1.0-RC2 YOU will get all the credit for the translation.

**Comment by 3d-gussner** [comment]
@nfsmedeiros Just flying over your translation is saw special characters ç and ã, that will not work with the LCD screen used by Prusa (has ROM A00 and not western euopean A02). Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?
+#define MSG_WATCH                           "Ecrã inform."
+#define MSG_RECTRACT                        "Retração"
+#define MSG_NO					"Não"

File: Firmware/language_pt.h [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+#define MSG_LOAD_ALL						"Carregar todos"
+
+
+#define MSG_RECTRACT                        "Retração"
+#define MSG_ERROR                        "ERRO:"
+#define(length=20) MSG_PREHEAT_NOZZLE       "Pre-aqueca bocal!"
+#define MSG_SUPPORT "Suporte"
+#define(length=20) MSG_CORRECTLY			"Troca correta?"
+#define MSG_YES					"Sim"
+#define MSG_NO					"Não"
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?

File: Firmware/language_pt.h [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+#define(length=17) MSG_LOAD_FILAMENT_4				"Carga filamento 4"
+#define(length=17) MSG_UNLOAD_FILAMENT_1				"Descarga filam. 1"
+#define(length=17) MSG_UNLOAD_FILAMENT_2				"Descarga filam. 2"
+#define(length=17) MSG_UNLOAD_FILAMENT_3				"Descarga filam. 3"
+#define(length=17) MSG_UNLOAD_FILAMENT_4				"Descarga filam. 4"
+#define MSG_UNLOAD_ALL						"Descarregar todos"
+#define MSG_LOAD_ALL						"Carregar todos"
+
+
+#define MSG_RECTRACT                        "Retração"
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?

File: Firmware/language_pt.h [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+#define MSG_FACTOR                          " \002 Fator"
+#define MSG_TEMPERATURE                     "Temperatura"
+#define MSG_MOTION                          "Movimento"
+#define MSG_VOLUMETRIC                      "Filamento"
+#define MSG_VOLUMETRIC_ENABLED		        "E em mm3"
+#define MSG_STORE_EPROM                     "Guardar na mem."
+#define MSG_LOAD_EPROM                      "Ler da memoria"
+#define MSG_RESTORE_FAILSAFE                "Rest. Val. Segur."
+#define MSG_REFRESH                         "\xF8" "Atualizar"
+#define MSG_WATCH                           "Ecrã inform."
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?

File: Firmware/language_pt.h [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+ *
+ * LCD Menu Messages
+ * Please note these are limited to 17 characters!
+ *
+ */
+
+#define(length=20) WELCOME_MSG              CUSTOM_MENDEL_NAME " pronta."
+#define MSG_SD_INSERTED                     "Cartao inserido"
+#define MSG_SD_REMOVED                      "Cartao removido"
+#define MSG_MAIN                            "Principal"
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: In https://github.com/MarlinFirmware/Marlin/blob/1.1.x/Marlin/language_pt.h it is "Menu principal" or is yours a common description?

File: Firmware/language_pt.h [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+#define MSG_DISABLE_STEPPERS                "Desativar motores"
+#define MSG_AUTO_HOME                       "Pos. de origem"
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: Compared to Marlin: "Ir para origem"

File: Firmware/language_pt.h [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+#define MSG_COOLDOWN                        "Arrefecer"
+#define MSG_SWITCH_PS_ON                    "Ligar alimentaçao"
+#define MSG_SWITCH_PS_OFF                   "Desligar aliment."
+#define MSG_MOVE_AXIS                       "Mover eixos"
+#define MSG_MOVE_X                          "Mover X"
+#define MSG_MOVE_Y                          "Mover Y"
+#define MSG_MOVE_Z                          "Mover Z"
+#define MSG_MOVE_E                          "Extrusora"
+#define MSG_SPEED                           "Velocidade"
+#define MSG_NOZZLE                          "Bocal"
```
Comment (3d-gussner) [inline Firmware/language_pt.h]: Marlin "Bico"?

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
@nfsmedeiros These are few things to review. I don't speak Portuguese just compared to Marlin PT translation.
Again thanks a lot.

**Comment by nfsmedeiros** [comment]
@3d-gussner 

Sorry about that.
I tried to check all the special characters and missed some I guess.
You can convert all of them

ã, á, â, etc -> a
é, ê, etc -> e
ç -> c

The meaning will not be affected.
For a portuguese/brazilian/african user there is no problem with that.
I can do that, no worries.

File: Firmware/language_pt.h (hunk 6 above)
Comment (nfsmedeiros) [reply Firmware/language_pt.h]: "Bico" has a special meaning in Portugal.
Roughly it is a X-Rated word... but yes, we can use it.
I'll edit that.

File: Firmware/language_pt.h (hunk 4 above)
Comment (nfsmedeiros) [reply Firmware/language_pt.h]: Main -> Principal
Main menu -> Menu Principal
But OK, we can use Menu Principal

**Review by 3d-gussner — APPROVED** [review]

**Comment by 3d-gussner** [comment]
@nfsmedeiros I removed the unused (length=xx, lines=yy) and changed the format a bit

**Pull request merged**